python/graphstorm/gsf.py

```diff
@@ -131,14 +131,16 @@ def create_builtin_node_gnn_model(g, config, train_task):
         Configurations
     train_task : bool
         Whether this model is used for training.
+    share_decoder: bool
+        Whether this model share decoder for different node types.
 
     Returns
     -------
     GSgnnModel : The GNN model.
     """
-    return create_builtin_node_model(g, config, train_task)
+    return create_builtin_node_model(g, config, train_task, share_decoder)
```

Comment:
If we can add `share_decoder` as part of config, then user can directly create this model by setting model arguments.

Reply:
solved